bitrise/print.go

```diff
@@ -361,6 +347,39 @@ func getDeprecateNotesRows(notes string) string {
 	return formattedNote
 }
 
+func getRow(str string) string {
+	str = stringutil.MaxLastCharsWithDots(str, stepRunSummaryBoxWidthInChars-4)
+	return fmt.Sprintf("| %s |", str+strings.Repeat(" ", stepRunSummaryBoxWidthInChars-len(str)-4))
+}
+
+// evaluatedVersion introduced for testing purposes, until StepInfoModel is updated to have the same property
+func buildUpdateRow(stepInfo stepmanModels.StepInfoModel, width int, evaluatedVersion string) string {
```

Comment:
evaluatedVersion is not required for testing now, StepInfoModel contains it from vendor for testing purposes

Reply:
the rewire for the model field is missing